app/packs/src/models/Element.js

```diff
@@ -44,6 +44,10 @@ export default class Element {
     return this.is_new == true
   }
 
+  set isNew(boolean) {
```

Comment:
See my question on InventoryTab line 74. I think the assignment there is redundant and as such the definition of the setter here is redundant as well